pkg/ui/workspaces/db-console/src/redux/timeScale.ts

```diff
@@ -167,6 +167,11 @@ export const selectTimeScale = createSelector(
   timeScaleState => timeScaleState.scale,
 );
 
+export const selectMetricsTime = createSelector(
```

Comment:
nit: Maybe we can remove the `createSelector` if it just returns the one field.

Reply:
Done.